src/range_proof.rs

```diff
@@ -12,209 +11,137 @@ use curve25519_dalek::ristretto::RistrettoPoint;
 use curve25519_dalek::scalar::Scalar;
 use curve25519_dalek::traits::IsIdentity;
 
+use generators::{Generators, GeneratorsView};
 use inner_product_proof::InnerProductProof;
-
 use proof_transcript::ProofTranscript;
-
 use util;
 
-use generators::GeneratorsView;
-
 /// The `RangeProof` struct represents a single range proof.
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct RangeProof {
     /// Commitment to the bits of the value
-    A: RistrettoPoint,
+    pub(crate) A: RistrettoPoint,
```

Comment:
what's the rationale for making this `pub(crate)`?

Reply:
Originally this was so that the aggregation code could use the internals of the RangeProof struct. It may no longer be necessary and after #84 I'm pretty sure it's not necessary, so that's a good catch. I think it's better to fix it there, though.

Reply:
Fixed in https://github.com/chain/ristretto-bulletproofs/pull/84/commits/0e7ba65fa0ed9a3706bc1e90b51c9d526e090997 , thanks!

Reply:
👍